text/0000-deprecation.md

```diff
@@ -0,0 +1,105 @@
+- Feature Name: Public Stability
+- Start Date: 2015-09-03
+- RFC PR: 
+- Rust Issue: 
+
+# Summary
+
+This RFC proposes to allow library authors to use a `#[deprecated]` attribute,
+with `since="`(version)`"`, `reason="`(free text)`"` and 
+`surrogate="`(text or surrogate declaration)`"` fields. The compiler can then
```

Comment:
<code>&lt;code&gt;since = "\*version\*"&lt;/code&gt;</code> produces <code>since = "*version*"</code> which typically looks better. (Also note the whitespace around the equals sign, which is conventional.)